ast/parameter.go

```diff
@@ -438,5 +438,5 @@ func (p *Parameter) getStorageLocationFromCtx(ctx *parser.ParameterDeclarationCo
 		}
 	}
 
-	return ast_pb.StorageLocation_MEMORY
+	return ast_pb.StorageLocation_DEFAULT
```

Comment:
I changed this from `Memory` to `Default` because we needed a way to print the storage location "memory" only when it is specified in the original antlr ast.

Reply:
We print "" on `Default`